liquibase-core/src/main/java/liquibase/util/SqlUtil.java

```diff
@@ -120,7 +123,7 @@ public static Object parseValue(Database database, Object val, DataType type) {
         }
 
         String typeName = type.getTypeName();
-        try (Scanner scanner = new Scanner(stringVal.trim())) {
+        try (Scanner scanner = new Scanner(stringVal.trim()).useLocale(US)) {
```

Comment:
@nvoxland, should we be hard-coding the locale in this try statement? If so, can you please explain why? Thank you!

Reply:
hi, @crizzis, thank you for the explanation. I didn't realize that `Scanner` was locale-sensitive. Were you on a machine with a different locale when you were coding the fix & saw a problem?